### ISSUE TITLE
Improve error message when date format is invalid

### ISSUE THREAD

**Issue by benvenker** [sourcegraph/sourcegraph#37788]

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by almeidapaulooliveira** [comment]
if it is hard to parse the inputted date in the info message, we could inform only the mask YYYY-MM-DD.

> Invalid date format. Please use the 'YYYY-MM-DD' format.

**Comment by camdencheek** [comment]
Just verified that this is still relevant
